Formula/ldc.rb

```diff
@@ -27,6 +27,22 @@ class Ldc < Formula
     sha256 "313b430f7066f800b0c9f99f57c17dbbec08650b4014058d4b011bac9bf67830" => :yosemite
   end
 
+  devel do
+    url "https://github.com/ldc-developers/ldc/releases/download/v1.2.0-beta1/ldc-1.2.0-beta1-src.tar.gz"
```

Comment:
Homebrew is scanning the version as `1.2.0`, so you'll need an explicit `version` specification.